.github/workflows/continuous-integration.yaml

```
@@ -90,6 +90,12 @@ jobs:
       - name: "Coverage Report"
         run: python .pipeline/scripts/print-coverage.py --jacoco-report-pattern "**/target/site/jacoco/jacoco.csv"
 
+      - name: "OWASP Dependency check"
+        run: mvn org.owasp:dependency-check-maven:10.0.4:check -DnvdApiKey=$NVD_API_KEY -DfailBuildOnCVSS=7 -DskipProvidedScope=true -DsuppressionFile=dependency-check-suppression.xml
```

```suggestion
        run: mvn org.owasp:dependency-check-maven:10.0.4:check -DnvdApiKey=$NVD_API_KEY -DfailBuildOnCVSS=7 -DskipProvidedScope=true
```

I tested: Without the `-DsuppresionFile` the false-positive warnings are logged and the step is still successful.

I assume it would only fail with CVSS>=7. In that case I would suggest to completely remove the suppression file.

If you have a good reason to keep it, please move it to a subdirectory, not root path.